ansible/playbooks/AdoptOpenJDK_AIX_Playbook/roles/maxuproc/tasks/main.yml

```diff
@@ -0,0 +1,7 @@
+#################################
+# Ensure maxuproc is set to 512 #
+#################################
```

Comment:
Perhaps add a link to the issue where the symptoms of the problem that this solves is? i.e. the resource messages